.github/workflows/docker-build-test-upload.yml

```
@@ -38,7 +38,7 @@ jobs:
             # Also, they might have running or stopped containers,
             # which are not cleaned up by `docker system prun`
             - name: Reset docker state and cleanup artifacts 🗑️
-              if: ${{ inputs.platform != 'x86_64' }}
+              if: ${{ startsWith(inputs.runsOn, 'ubuntu') }}
```

The new changes only support `ubuntu`?

---

This step of resetting Docker is only needed on our self-hosted arm64 runner. This if statement just ensures that this step is skipped run on the official x86 runner.

In a next PR I am going to remove this altogether since we want to stop using the self-hosted runner.

---

Ah, made a mistake here, the if condition is inverted, it should be 

```suggestion
              if: ${{ ! startsWith(inputs.runsOn, 'ubuntu') }}
```

This is fixed in #764